azurerm/resource_arm_virtual_network_gateway_test.go

```diff
@@ -269,11 +276,20 @@ resource "azurerm_virtual_network_gateway" "test" {
     private_ip_address_allocation = "Dynamic"
     subnet_id = "${azurerm_subnet.test.id}"
   }
+
+  vpn_client_configuration {
```

Comment:
Could we add an entirely new test for these properties instead of adding them to an existing test?

Reply:
sure, done.